canonical/ubuntu.com
# Wd 10982 pro copy updates

## Done

- Updated Pricing page
- Updated Release cycle page
- Updated Ubuntu Pro description

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Checkout [/pricing/pro](https://ubuntu-com-13855.demos.haus/pricing/pro)
    - Make sure it matches with [the copy doc](https://docs.google.com/document/d/1Ynk0iLa9hjrdich6ATTsybtPbDwW_k4vRI0nk_HNhcI/edit)
- Checkout [/about/release-cycle](https://ubuntu-com-13855.demos.haus/about/release-cycle)
    - Make sure it matches with [the copy doc](https://docs.google.com/document/d/1-l3B7AoyDoETaL80UhlfEWCVKmMWPdy3zoF5LpsGcYQ/edit)
- Checkout [/pro/subscribe](https://ubuntu-com-13855.demos.haus/pro/subscribe)
    -  Select any product and proceed to checkout
    - Open the [ubuntu service description](https://ubuntu-com-13855.demos.haus/legal/ubuntu-pro-description) at the bottom of the page
    - Should match with the [copy](https://docs.google.com/document/d/15URkXf2vh7bv89n-cWQPeaP2EehSNUnUCoZvGbNTYFA/edit)

## Issue / Card

Fixes [#WD-10982](https://warthogs.atlassian.net/browse/WD-10982)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
